contracts/optimistic-ethereum/OVM/verification/OVM_StateTransitioner.sol

```diff
@@ -24,6 +24,17 @@ import { OVM_FraudContributor } from "./OVM_FraudContributor.sol";
 
 /**
  * @title OVM_StateTransitioner
+ * @dev The OVM_StateTransitioner coordinates the execution of a state transition during the evaluation of a 
+ * fraud proof. It controls the OVM_ExecutionManager, as well as an instance of the OVM_StateManager (which is 
+ * uniquely created for each fraud proof).
+ * Once a fraud proof has been initialized, this contract is populated with the pre-state root as
+ * well as the OVM storage slots used in the transaction whose state commitment is being proven fraudulent.
```

Comment:
To disambiguate, I would indicate that it is specifically the SM which gets populated with the OVM storage slots.  This contract authenticates/verifies that those populations check out with the `preStateRoot`, but it does not store them itself.